nuxeo-features/nuxeo-platform-notification/nuxeo-platform-notification-web/src/main/java/org/nuxeo/ecm/webapp/notification/SubscriptionsAction.java

```diff
@@ -230,4 +232,13 @@ public void setInheritedNotifications(List<Notification> inheritedNotifications)
         this.inheritedNotifications = inheritedNotifications;
     }
 
+    /**
+     * @since 9.2
+     */
+    public boolean canFollow() {
+        DocumentModel currentDoc = navigationContext.getCurrentDocument();
+        PublisherService publisherService = Framework.getService(PublisherService.class);
+        return !currentDoc.isProxy() && !publisherService.isPublishedDocument(currentDoc);
```

Comment:
We don't need the `PublisherService` check here (which is likely redundant). The important thing to fix is that we don't want to do writes on a proxy document (pointing to a version or live doc).